evm_integration.py

```diff
@@ -39,6 +38,8 @@ def main():
     adapter_factory_contract: {latest_env['adapterFactory']}
     legacy_tx: false
     poll_interval: 1
+  solana:
+    express_relay_program_id: GwEtasTAxdS9neVE4GPUpcwR7DB7AizntQSPcG36ubZM
```

Comment:
@anihamde @m30m What do you thing about this part?
I couldn't find a resources file for svm tilt setup. In other files we used the actual variable and didn't read the contract address from any kind of generated config file. Should I continue with it or it's easy to create a config file from the generated accounts and addresses? (Like what we have for evm)

Reply:
The solana program addresses are being set in the `test-validator-params.sh` file. they don't have to be set there, we can add those as arguments to the call in tilt, but if so, we should create a separate `svm_integration.py` file and `local_resource` to create env vars for express relay and dummy program addresses.

Reply:
Thank you Ani.
As this value is not going to be updated, I will continue with the current version.

Reply:
for code quality purposes, we should probably ensure we are using the same program ID across the `test-validator-params.sh` file and here. probably easiest would be to just read what's in the `test-validator-params.sh` file here